### PR TITLE
add kubernetes-policy-controller chart

### PR DESCRIPTION
**Pull request by rpahli**
#### What this PR does / why we need it:

this PR add a helm chart for https://www.openpolicyagent.org/.
This helm chart is helpful to manage cluster policies 
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/kubernetes-policy-controller/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes Policy Controller
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/Chart.yaml]: What does a k8s policy controller?

File: charts/kubernetes-policy-controller/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+1. Get the application URL by running these commands:
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/NOTES.txt]: Looks empty...

File: charts/kubernetes-policy-controller/templates/clusterrolebinding.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-opa-viewer
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/clusterrolebinding.yaml]: Whats about the HELM labels?

File: charts/kubernetes-policy-controller/templates/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-kubernetes-matches
+  namespace: {{ .Release.Namespace }}
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/configmap.yaml]: Whats about the HELM labels?

File: charts/kubernetes-policy-controller/templates/configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+    matches[[kind, namespace, name, resource]] {
+      resource := kubernetes[kind][namespace][name].object
+    }
+
+    matches[[kind, namespace, name, resource]] {
+        resource := kubernetes[kind][namespace][name]
+    }
+---
+apiVersion: v1
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/configmap.yaml]: It's not a common practice to have multiple resources specified in a single manifest.

File: charts/kubernetes-policy-controller/templates/configmap.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-kubernetes-policymatches
+  namespace: {{ .Release.Namespace }}
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/configmap.yaml]: Whats about the HELM labels?

File: charts/kubernetes-policy-controller/templates/role.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ .Release.Name }}-configmap-modifier
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/role.yaml]: Whats about the HELM labels?

File: charts/kubernetes-policy-controller/templates/rolebinding.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ .Release.Name }}-configmap-modifier
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/rolebinding.yaml]: Whats about the HELM labels?

File: charts/kubernetes-policy-controller/templates/webhookconfiguration.yaml [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+      caBundle: {{ b64enc $ca.Cert }}
+{{ else }}
+      caBundle: {{ b64enc .Values.admissionControllerCA }}
+{{ end }}
+      service:
+        name: {{ template "kubernetes-policy-controller.fullname" . }}
+        namespace: {{ .Release.Namespace }}
+        path: "/v1/admit"
+---
+apiVersion: v1
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/templates/webhookconfiguration.yaml]: It's not a common practice to have multiple resources specified in a single manifest.

File: charts/kubernetes-policy-controller/Chart.yaml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/Chart.yaml]: Mhh... you're specifying OPA v.0.6 and kubernetes-policy-controller v.1.2 in your default values.yaml.

**Comment by axdotl** [comment]
Please fill in the PR template.

File: charts/kubernetes-policy-controller/templates/webhookconfiguration.yaml (hunk 9 above)
Comment (rpahli) [reply charts/kubernetes-policy-controller/templates/webhookconfiguration.yaml]: I used some variables which I need for both resources

File: charts/kubernetes-policy-controller/Chart.yaml [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+description: A Helm chart for Kubernetes Policy Controller
+name: kubernetes-policy-controller
+version: 0.1.0
+home: https://github.com/kiwigrid/helm-charts
+sources:
+- https://github.com/open-policy-agent/kubernetes-policy-controller
+- https://github.com/open-policy-agent/opa
+maintainers:
+- name: rpahli
+  email: rico.pahlisch@kiwigrid.com
```
Comment (axdotl) [inline charts/kubernetes-policy-controller/Chart.yaml]: Please add an icon as well.

**Pull request merged**